XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/MixedRealityControllerMapping.cs

```diff
@@ -7,8 +7,9 @@
 using XRTK.Definitions.Devices;
 using XRTK.Definitions.Utilities;
 using XRTK.Interfaces.Providers.Controllers;
+using XRTK.Providers.Controllers;
 
-namespace XRTK.Providers.Controllers
```

Comment:
This one should keep the Providers namespace, since it's in a Providers/Controllers right?

Reply:
Great catch